tools/rna_tools/vienna_rna/rna2dfold.xml

```diff
@@ -16,63 +18,96 @@
         #if $varExists('$pfselect.pfscaling')
             --partfunc --pfScale=$pfselect.pfscaling
         #end if
-        #if $varExists('$advancedOptions.noconversion')
-            #if $advancedOptions.noconversion
-                --noconv
-            #end if
-            #if $advancedOptions.nogu
-                --noGU
-            #end if
-            #if $advancedOptions.noclosinggu
-                --noClosingGU
-            #end if
-            #if $advancedOptions.notetra
-                --noTetra
-            #end if
-            --maxDist1=$maxK
-            --maxDist2=$maxL
+        $noBT
+
+        $advancedOptions.noconversion
+		$advancedOptions.noclosinggu
+		$advancedOptions.circular
+    
+        #if $varExists('$advancedOptions.maxK')    
+            --maxDist1=$advancedOptions.maxK
+            --maxDist2=$advancedOptions.maxL
         #end if
+
         < $custom_input > $out_file
 ]]>
     </command>
     <inputs>
         <param format="txt,dbn" name="custom_input" type="data" label="Custom File"/>
         <param name="temperature" type="float" value="37.0" label="Temperature [°C]" help="-T"/>
-        <param name="nrbt" type="integer" value="1" label="number of backtrack of Boltzmann samples" help="--stochBT=INT"/>
-        <param name="nk" type="integer" value="-1" label="k distance to first reference structure of neighborhood in which to backtrack" help="--neighborhood=k:l"/>
-        <param name="nl" type="integer" value="-1" label="l distance to second reference structure of neighborhood in which to backtrack" help="--neighborhood=k:l"/>
-        <param name="dangling" type="select" label="how to treat dangling end energies" help="-d">
-            <option value="2" selected="true">unpaired bases participate in all dangling ends (2)</option>
-            <option value="0">ignore dangling ends (0)</option>
-            <option value="1">unpaired bases participate in one dangling end only (1)</option>
-            <option value="3">allow coaxial stacking (3)</option>
+        <param name="nrbt" type="integer" value="1" label="Number of backtrack of Boltzmann samples" help="--stochBT=INT"/>
```

Comment:
If you migrating the tools please have a look at the `argument` attribute: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool|inputs|param